_data/validator-libraries-modern.yml

```diff
@@ -3,63 +3,75 @@
   implementations:
     - name: Json.NET Schema
       url: https://www.newtonsoft.com/jsonschema
+      date-draft:
```

Comment:
Wait why not just `draft: ["2019-09", 7, 6, 4, 3]` ?

Reply:
LOL try it 😛

Reply:
@philsturgeon you have to make something somewhere more complicated, this was the easiest option in the <10 minutes I was willing to spend on it.